client/my-sites/themes/themes-header.jsx

```diff
@@ -20,6 +21,7 @@ import './themes-header.scss';
 const ThemesHeader = () => {
 	return (
 		<div className="themes__header">
+			<ScreenOptionsTab />
 			<FormattedHeader
```

Comment:
Is the `hasScreenOptions` prop missing here?

Reply:
Seems to be missing also here:

https://github.com/Automattic/wp-calypso/blob/64d9c606db7367681ebbecb4fa71c15d61d55f76/client/my-sites/types/main.jsx#L52-L58